addons/headless/functions/fnc_handleConnectHC.sqf

```diff
@@ -28,5 +28,8 @@ if (GVAR(log)) then {
     INFO_1("Registered HC: %1",_headlessClient);
 };
 
+//Exit if AI distribution is disabled
```

Comment:
Space after `//`.

Reply:
In my master it is fixed...but why isn't the commit added to the PR? o.O
Am I too dumb for github?
https://github.com/specop0/ACEX/commits/master

Reply:
Interesting. 🤔